### PR TITLE
8990 flags for opinion search

### PR DESCRIPTION
**Pull request by TomElliottFlexion**
Draft to run tests and make notes

### CODE REVIEW

File: check-env-variables.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
```
Comment (TomElliottFlexion) [inline check-env-variables.sh:1]: @codyseibert Can this be merged as part of this story?

File: web-client/src/presenter/computeds/FeatureFlags/featureFlagHelper.js [hunk 2]
```diff
@@ -18,12 +23,9 @@ export const featureFlagHelper = (get, applicationContext) => {
     );
   }
 
-  const isOpinionSearchEnabled = applicationContext.isFeatureEnabled(
-    'advanced_opinion_search',
-  );
-
   return {
-    isOpinionSearchEnabled,
+    isInternalOpinionSearchEnabled,
```
Comment (TomElliottFlexion) [inline web-client/src/presenter/computeds/FeatureFlags/featureFlagHelper.js:27]: Note: do we need to check role as well?

File: web-client/src/views/AdvancedSearch/AdvancedSearch.jsx [hunk 3]
```diff
@@ -113,12 +113,12 @@ export const AdvancedSearch = connect(
                 <DocumentSearchResults />
```
Comment (TomElliottFlexion) [inline web-client/src/views/AdvancedSearch/AdvancedSearch.jsx:113]: Potentially update to show both notifications when both order and opinion search are toggled off

File: check-env-variables.sh (hunk 1 above)
Comment (codyseibert) [reply check-env-variables.sh:1]: I'm not sure how this even got in this PR.  I made a PR to develop when I did this work.  Maybe I accidentally checked off the wrong branch?

**Pull request closed without merging**